### PR TITLE
{lang}[system/system] Java v13.0.2

### PR DESCRIPTION
**Pull request by Falkor**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-005 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/65f9c93f9e598c7038918dd622ea6572 for a full test report.

**Comment by lexming** [comment]
@Falkor I re-triggered the tests

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node352.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/25b9d1f34281c7ac8f794115d4534167 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node376.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c27da2ced9664e133b7d0f3486c3282c for a full test report.

File: easybuild/easyconfigs/j/Java/Java-13.0.2.eb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name = 'Java'
+version = '13.0.2'
+
+homepage = 'https://openjdk.java.net'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://download.java.net/java/GA/jdk%(version)s/d4173c853231432d94f001e99d882ca7/8/GPL/']
```
Comment (lexming) [inline easybuild/easyconfigs/j/Java/Java-13.0.2.eb:10]: The download URL does not work for me, I'm always prompted to accept Oracle's license. Is that not the case for you?
```suggestion
# Downloads are subject to license.
# Available from https://www.oracle.com/java/technologies/javase-jdk13-downloads.html
```

File: easybuild/easyconfigs/j/Java/Java-13.eb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Contribution from the Crick HPC team / UL HPC Team
+# uploaded by J. Sassmannshausen
+# updated to version 202
```
Comment (lexming) [inline easybuild/easyconfigs/j/Java/Java-13.eb:3]: I would update this comment as well to reflect the current version of this update.

File: easybuild/easyconfigs/j/Java/Java-13.0.2.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/j/Java/Java-13.0.2.eb:10]: I was mistaken, current `source_urls` do work to download OpenJDK. I was trying to make it work with Oracle's JDK which is not the intention. Sorry for the noise.

**Comment by lexming** [comment]
Going in, thanks @Falkor !

**Pull request merged**